redfish-core/lib/log_services.hpp

```diff
@@ -4993,15 +4993,18 @@ inline void fillSystemHardwareIsolationLogEntry(
                     {
                         entryJson["Severity"] = "Critical";
                     }
-                    else if ((*severity == "xyz.openbmc_project."
+                    else if (*severity == "xyz.openbmc_project."
                                            "HardwareIsolation."
-                                           "Entry.Type.Warning") ||
-                             (*severity == "xyz.openbmc_project."
-                                           "HardwareIsolation."
-                                           "Entry.Type.Manual"))
+                                           "Entry.Type.Warning")
                     {
                         entryJson["Severity"] = "Warning";
                     }
+                    else if (*severity == "xyz.openbmc_project."
+                                           "HardwareIsolation."
+                                           "Entry.Type.Manual")
```

Comment:
Seems weird Manual is a Severity...

Reply:
I would expect a severity of OK here as a severity

Reply:
agree

Reply:
For now we are not going to make any changes to the severity wording as the impact is big. Now fixed the CI failure please take a look.